src/main/java/at/hannibal2/skyhanni/config/features/About.java

```diff
@@ -11,7 +11,7 @@ public class About {
 
     @ConfigOption(name = "Current Version", desc = "This is the SkyHanni version you are running currently")
     @ConfigVersionDisplay
-    public Void currentVersion = null;
+    public transient Void currentVersion = null;
```

Comment:
What does transient do in this context?

Reply:
explicitly not serialized (supresses the warning for missing serialization)